cyclonedx-lib/build.xml

```diff
@@ -130,4 +259,3 @@
 		</sequential>
 	</macrodef>
 </project>
```

Comment:
Add this at the end of this file, so Linter jscpd ignores this file
```
<!-- jscpd:ignore-end -->
```

Reply:
@andrew-m-leonard added the lines in the correct place?

Reply:
Still looks like it's failing checks, so I'd imagine it needs to be the final line in the file after `</project>`.

Reply:
@sxa Still failing the check tho.